site-manage/site-manage-tool/tool/src/java/org/sakaiproject/site/tool/SiteAction.java

```diff
@@ -12053,6 +12053,9 @@ private void addNewSite(ParameterParser params, SessionState state) {
 					rp.addProperty(TEMPLATE_USED, templateSite.getId());
 				}
 
+				// Enable MathJax, if applies
+				MathJaxEnabler.prepareMathJaxForNewSite(site, state, (List<String>) state.getAttribute(STATE_TOOL_REGISTRATION_SELECTED_LIST));
```

Comment:
If enabled at system level, it will enable it for all tools, honoring the "enabled by default for the entire site" premise ... , @bjones86 is right, this should work better as its own property rather than "if enabled at system level enable on creation for all sites->tools"
I will add the change shortly, thanks!
F